### ISSUE TITLE
pcap import fails on Windows with per-machine install

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2715]
Repro is with [Zui v1.0.0](https://github.com/brimdata/zui/releases/tag/v1.0.0). This issue was first reported by a community user in a [Slack thread](https://brimdata.slack.com/archives/C010DR0HHMF/p1678177258353039).

The attached video shows a Windows 10 environment where Zui was installed with the "per-machine" option that was added in #2686. When a pcap is imported, an error pops up:

```
Unable to generate full summary logs from PCAP 
Detail: suricatarunner.exe exited with code 1
stdout: (no output)
stderr:
2023/03/07 19:36:53 makeConfig failed: open C:\Program Files\Zui\resources\app.asar.unpacked\zdeps\suricata\brim-conf-run.yaml: Access is denied.
```
 
https://user-images.githubusercontent.com/5934157/223613962-43f27cb4-6f76-4ecf-ae9b-bbca9ee19638.mp4

I think I see what the problem is. The Windows [suricataupdater.exe](https://github.com/brimdata/build-suricata/blob/master/go/updater/main.go) makes use of the `BRIM_SURICATA_USER_DIR` environment variable that's sent down from Zui here:

https://github.com/brimdata/zui/blob/904edcf49b8cf8e2ee33bd58fbbf480881532231/src/plugins/brimcap/brimcap-plugin.ts#L78

...wheres with [suricatarunner.exe](https://github.com/brimdata/build-suricata/blob/master/go/runner/main.go) it seems like the creation of the `brim-conf-run.yaml` happens under the "zdeps" directory near where the binaries live and are intended to be read-only (https://zui.brimdata.io/docs/support/Filesystem-Paths#application-binaries). I can also see that the shell-script-based runner scripts we use on Linux/macOS do seem to make use of `BRIM_SURICATA_USER_DIR` so they end up writing their `brim-conf-run.yaml` in the separate read/write "user data" area (https://zui.brimdata.io/docs/support/Filesystem-Paths#user-data).

In conclusion, I think the `suricatarunner.exe` needs some enhancement to make use of the `BRIM_SURICATA_USER_DIR`.

**Comment by philrz** [comment]
Verified with [Zui Insiders 1.0.1-9](https://github.com/brimdata/zui-insiders/releases/tag/v1.0.1-9) which includes this fix.

As shown in the attached video, now on a "per machine" install where the app binaries land in `%ProgramFiles%\Zui`, pcap import works now whereas it failed in the past. Another problem was fixed at the same time that's also shown in the video: Previously on Windows, the Suricata rule set that shipped with the app binaries was always being used, but now the updated one below the user data directory is used. This was tested by manually hacking a rule in `%APPDATA%\Zui - Insiders\data\suricata\rules\suricata.rules` that would be triggered by the pcap, then re-importing the pcap and observing that the hack is present.

https://user-images.githubusercontent.com/5934157/226024383-ab7abdcc-cfa3-4040-9828-e197cff30164.mp4

Thanks @nwt!
